common4j/src/main/com/microsoft/identity/common/java/platform/AbstractDevicePopManager.java

```diff
@@ -1010,8 +1032,19 @@ private String mintSignedHttpRequestInternal(@Nullable final String httpMethod,
         throw clientException;
     }
 
+    private static boolean isNegativeInternalError(@NonNull final Throwable t) {
+        if ((t.getMessage() != null && t.getMessage().contains(NEGATIVE_THIRTY_THREE_INTERNAL_ERROR)) ||
+                (t.getCause() != null && t.getCause().getMessage().contains(NEGATIVE_THIRTY_THREE_INTERNAL_ERROR))
```

Comment:
I don't think you need to check for cause in this method anymore since the caller is now already check for all three layers

Reply:
updated